### PR TITLE
Use dedicated tests user

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

File: tests/API/APIRestTest.php [hunk 1]
```diff
@@ -568,8 +571,8 @@ public function testGetItem($session_token, $computers_id) {
       $this->assertFalse(is_numeric($data['entities_id'])); // for expand_dropdowns
       $this->assertArrayHasKey('_logs', $data); // with_logs == true
 
-      // Get the root-entity
-      $res = $this->doHttpRequest('GET', 'Entity/0',
+      // Get user's entity
+      $res = $this->doHttpRequest('GET', 'Entity/1',
```
Comment (remicollet) [inline tests/API/APIRestTest.php]: Shouldn't you retrieve entity number from test set ?

Comment (remicollet) [inline tests/API/APIRestTest.php]: `getItemByTypeName('Entity', '_test_root_entity', true);`

File: tests/API/APIRestTest.php [hunk 2]
```diff
@@ -276,14 +277,15 @@ public function testGetFullSession($session_token) {
      * @depends testInitSessionCredentials
      */
    public function testGetMultipleItems($session_token) {
-      // Get the User 'glpi' and the root entity in the same query
+      // Get the User TU_USER and the entity in the same query
+      $uid = User::getIdByName(TU_USER);
```
Comment (remicollet) [inline tests/API/APIRestTest.php]: Or  getItemByTypeName('User', 'TU_USER',  true);

Comment (trasher) [inline tests/API/APIRestTest.php]: Id don't know what is the best

File: tests/API/APIRestTest.php (hunk 1 above)
Comment (trasher) [inline tests/API/APIRestTest.php]: Oh.. ok, will change :)

**Pull request merged**